src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandContext.java

```diff
@@ -224,4 +274,19 @@ public SmallRyeConfig getConfig() {
     }
     return ConfigProvider.getConfig().unwrap(SmallRyeConfig.class);
   }
+
+  public OperationsConfig operationsConfig() {
+    if (operationsConfig != null) {
+      return operationsConfig;
+    }
+    // During tests:
+    return getConfig(OperationsConfig.class);
+  }
+
+  public FilterClause resolveFilterClause(FilterSpec filter) {
+    if (filter == null) {
+      return null;
+    }
+    return FilterClauseDeserializer.INSTANCE.buildFilterClause(this, filter.json());
```

Comment:
@amorton Only temporarily hard-coded, will branch out to schema-specific deserializers next.